packages/redux/src/settings/reducer/settings.ts

```diff
@@ -0,0 +1,69 @@
+import * as actionTypes from '../actionTypes.js';
+import { type AnyAction, combineReducers, type Reducer } from 'redux';
+import type { BlackoutError } from '@farfetch/blackout-client';
+import type { SettingsState } from '../types/index.js';
+
+export const INITIAL_STATE: SettingsState = {
+  error: null,
```

Comment:
Same as https://github.com/Farfetch/blackout/pull/922/files#r1407387185

Reply:
Done, splitting by the query